setup.py

```diff
@@ -55,5 +55,10 @@ def get_dependencies():
     install_requires=get_dependencies(),
     entry_points={
         'console_scripts': ['cddl-gen=cddl_gen.cddl_gen:main'],
-    }
+    },
+    data_files=[
+        ("lib/cddl-gen/include",
+            ("include/cbor_decode.h", "include/cbor_encode.h", "include/cbor_common.h",
+                "include/cbor_debug.h")),
+        ("lib/cddl-gen/src", ("src/cbor_decode.c", "src/cbor_encode.c", "src/cbor_common.c"))]
```

Comment:
AFAICT these will not be delivered with the installation. I have tried a few different things locally, but I haven't found a way to be able to include these in the installation folder alonside cddl-gen without being _in_ the `cddl-gen` package directory that contains `cddl_gen.py`. I think we had a similar issue with `VERSION` in #88 .

Aren't these necessary for delivery given the `is_relative_to` functions earlier in this file?

Reply:
They appear outside the tree.

```
$ ls ~/.local/lib/python3.9/site-packages/cddl_gen
cddl_gen.py  __init__.py  __pycache__  VERSION
$ ls ~/.local/lib/cddl-gen/
include  src
```